users/d4mation/config.h

```diff
@@ -0,0 +1,13 @@
+#define NO_ACTION_MACRO
+#define NO_ACTION_FUNCTION
+
+#define DISABLE_LEADER
```

Comment:
This is no longer needed, actually.  It's now disabled by default and requires `LEADER_ENABLE = yes` to enable.